IntelBluetoothFirmware.xcodeproj/project.pbxproj

```diff
@@ -466,19 +448,17 @@
 			isa = XCBuildConfiguration;
 			buildSettings = {
 				ARCHS = x86_64;
+				CODE_SIGNING_ALLOWED = NO;
 				CODE_SIGN_STYLE = Manual;
```

Comment:
Replacing both `CODE_SIGNING_ALLOWED = NO;` and `CODE_SIGN_STYLE = Manual;` with `CODE_SIGN_IDENTITY = "";` Same reason.

Reply:
No, it will not generate `_CodeSignature` anymore since code signing is completely disabled by https://github.com/OpenIntelWireless/IntelBluetoothFirmware/blob/6ac888a3f9ee0e1053f5d4c8356ab1120f6e09cb/IntelBluetoothFirmware.xcodeproj/project.pbxproj#L451